retrieval/raw/raw_endpoint.go

```diff
@@ -43,19 +49,38 @@ func (c *Client) createRequest(method, url string, body io.Reader) (*http.Reques
 }
 
 //GetRawEndPoint retrives the body of HTTP response from a given sourcename, working directory, and URL
-func (c *Client) GetRawEndPoint(sourceName string, workDir *os.File, URL string) (rawRespFile *os.File, rerr error) {
+func (c *Client) GetRawEndPoint(sourceName string,
+	workDir *os.File, URL string) (rawRespFile *os.File, err error) {
+
+	attempts := c.retries + 1
+
+	for i := uint(0); i < attempts; i++ {
+		rawRespFile, err = downloadToFile(c, sourceName, workDir, URL, i)
+		if err != nil {
+			log.Printf("%v URL: %s retrying: %v", err, URL, i)
+			time.Sleep(time.Duration(int64(math.Pow(2, float64(i)))) * time.Second)
+			continue
```

Comment:
kinda minor, but on the last "attempt", this will still wind up doing another sleep, then just returns (b/c it bounces out of the for loop), just an extra, unnecessary sleep. another option would be to move the sleep to the top of the for loop, and just always sleep `if i>0`, conceptually waiting to make the call on all but the first attempt.